framework/include/wifi_manager/wifi_manager.h

```diff
@@ -67,6 +67,14 @@ typedef enum {
 	WIFI_MANAGER_ALREADY_CONNECTED,
 } wifi_manager_result_e;
 
+/**
+ * @brief Wi-Fi disconnect event reason
+ */
+typedef enum {
+	WIFI_MANAGER_DISCONNECT,
+	WIFI_MANAGER_RECONNECT, //AUTOCONNECT
+} wifi_manager_disconnect_to;
```

Comment:
you have to use _e instead of _to; it's conventional for example wifi_manager_result_e

Reply:
Thanks you for informing. The code now follows the convention.
Modification completed.